stable/admin/templates/configmap.yaml

```diff
@@ -41,4 +41,17 @@ metadata:
     release: {{ .Release.Name | quote }}
   name: {{ template "admin.fullname" . }}-nuoadmin
 data:
-{{ (.Files.Glob "files/nuoadmin").AsConfig | indent 2 }}
\ No newline at end of file
+{{ (.Files.Glob "files/nuoadmin").AsConfig | indent 2 }}
+---
+apiVersion: v1
```

Comment:
whats this configmap for?

Reply:
This config map allows any container to find the address of the zeroeth (admin...-0) pod - hereafter AP-0

So...
I want to push logging messages to the admin layer, using the PUT diagnostics/log endpoint.
However, those log lines are only stored by the admin that received them. So if you send them via the k8s service, the messages will get scattered randomly across all APs in the domain.

This configmap allows the container to address the AP-0 container directly, so that all the log messages are sent to the same AP.
This solution is not perfect:
* PRO: The AP-0 is the first to be started, and the last to be shut down, so it is more likely to be there than any other AP.
* PRO: All such messages can ALWAYS be found in the AP-0 logs;
* CON: if the AP-0 has some problem, it is possible for the admin layer to be up, but the AP-0 to be unavailable.
   ... However, this is also true if we used a different algorithm that identified the AP that this engine pod used as its peer

The only other viable algorithm I could think of is to ping the admin ipcluster service, and capture the returned address.
However, that returns an address and not a hostname, so if the AP restarts during the process, then the logging would fail and be lost.

Reply:
I believe that @db2882 and @adriansuarez want to unify what nuodocker does and send all operations to a single admin instance.